core/data/src/main/java/org/mifos/mobilewallet/core/data/fineract/entity/mapper/ClientDetailsMapper.kt

```diff
@@ -0,0 +1,30 @@
+package org.mifos.mobilewallet.core.data.fineract.entity.mapper
+
+import com.mifos.mobilewallet.model.entity.client.Client
+import javax.inject.Inject
+
+/**
+ * Created by naman on 10/7/17.
+ */
+class ClientDetailsMapper @Inject constructor() {
+    fun transformList(clients: List<Client?>?): List<com.mifos.mobilewallet.model.domain.client.Client> {
```

Comment:
use kotlin alias here and import domain client like 
`import com.mifos.mobilewallet.model.domain.client.Client as DomainClient`